lib/astutils.h

```diff
@@ -178,7 +178,7 @@ bool isTemporary(bool cpp, const Token* tok, const Library* library, bool unknow
 const Token* previousBeforeAstLeftmostLeaf(const Token* tok);
 Token* previousBeforeAstLeftmostLeaf(Token* tok);
 
-const Token * nextAfterAstRightmostLeaf(const Token * tok);
+CPPCHECKLIB const Token * nextAfterAstRightmostLeaf(const Token * tok);
```

Comment:
I am not strongly against this but it does not seem related to this PR?

Reply:
`testrunner` is linked against `cppcheck-core.dll`. But since these objects were not exported it explicitly compiled in `astutils.cpp` which was obviously a hack.